pkg/utils/utils.go

```diff
@@ -29,9 +30,13 @@ const (
 // Ensures that the generated chain name is exactly
 // maxChainLength chars in length
 func FormatChainName(name string, id string) string {
-	chainBytes := sha512.Sum512([]byte(name + id))
-	chain := fmt.Sprintf("%s%x", chainPrefix, chainBytes)
-	return chain[:maxChainLength]
+	return FormatChainNameWithPrefix(name, id, "")
```

Comment:
Could you update there docstring (and also add one to next func) according to https://github.com/golang/go/wiki/CodeReviewComments#doc-comments and https://golang.org/doc/effective_go.html#commentary
Especially I'm asking about:
```
If every doc comment begins with the name of the item it describes, you can use the doc subcommand of the go tool and run the output through grep. Imagine you couldn't remember the name "Compile" but were looking for the parsing function for regular expressions, so you ran the command,

$ go doc -all regexp | grep -i parse
```